tools/super-tox.py

```diff
@@ -348,6 +369,8 @@ def get_lxd_instance(name: str, image_alias: str, create: bool = False):
 
     If create is True, then if there is no matching instance, then create one
     using the provided image alias."""
+    import pylxd  # type: ignore
+
```

Comment:
FWIW, my plan is to rip this whole thing out (I opened an issue for that now). Wrong solution for the problem, I think.